draft-ietf-quic-transport.md

```diff
@@ -526,7 +526,7 @@ an unopened stream indicates that the remote peer no longer wishes to receive
 data on this stream.  Either frame might arrive before a STREAM or
 STREAM_DATA_BLOCKED frame if packets are lost or reordered.
 
-Before creating a stream, all streams of the same type with lower-numbered
+Before a new stream is created, all streams of the same type with lower-numbered
```

Comment:
Maybe we don't have to clearly say "new".

Reply:
Yeah, I don't think we need "new" in the sentence.

Reply:
Thanks! Fixed it.